pages/05.android/06.solving-problems/06.low-level-settings/docs.en.md

```diff
@@ -35,12 +34,26 @@ Nowadays, more and more often SMS and MMS are sent via network, and when they ar
 
 **pref.samsungpay.autopause.enable**
 
-This setting was developed specifically for the South Korean users who used to face the following problem — the *“Samsung Pay”* service doesn’t work on devices with running VPN. AdGuard, in its turn, uses Local VPN to filter traffic and this mode is on by default. 
+This setting was developed specifically for the South Korean users who used to face the following problem — the *"Samsung Pay"* service doesn’t work on devices with running VPN. AdGuard, in its turn, uses Local VPN to filter traffic and this mode is on by default. 
```

Comment:
You know what, let's extract it in a separate article and give a link to that article here.

This way we'll be able to link that article from the app.